### DISCUSSION TITLE
Version 2.1.0 board suggestions

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#974] [General]
Hi 
Nice gift for Christmas !!! 🎅🏻🎄Thanks you very much 👍🏻

Now with the Ethernet connectivity we will have a much more stable connection !

Do you suggest any board ?

Thans again and have a nice end of year ! 🥂

**Comment by HomeSpan** [comment]
Thanks and happy new year as well!

Since most of my kits are based on the Adafruit Featherboard layout, I've used this https://www.adafruit.com/product/3201 with good success.  It's based on a W5500 (using the SPI bus), which is directly supported by the Arduino-ESP32 library.

**Reply by Sill1000** [reply]
Thanks for your answer !

It seems that we canot find any ESP32 board with the Ethernet connector.

The adafruit is the solution. We can piggy bag it directly on any ESP32 board.
